extension/api.js

```diff
@@ -0,0 +1,80 @@
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "skeleton" }]*/
+/* global ExtensionAPI */
+
+const PREF_HOTFIX_DONE = "extensions.hotfix-intermediate-2018.done";
```

Comment:
To avoid an unnecessary pref and unnecessary certificate reverification on the next update, could we use `extensions.signatureCheckpoint` ? If set, return early (implies patched Firefox version), if not set and completed, set value to 1.

This pref name is used at https://phabricator.services.mozilla.com/D242193